tests/test/faaslet/test_flushing.cpp

```diff
@@ -58,22 +58,25 @@ TEST_CASE_METHOD(FlushingTestFixture,
     loader.uploadSharedFile(fileName, fileBytes);
 
     // Check that the underlying shared file is in place
-    boost::filesystem::path sharedPath(conf.sharedFilesDir);
+    std::filesystem::path sharedPath(conf.sharedFilesDir);
     sharedPath.append(fileName);
-    REQUIRE(boost::filesystem::exists(sharedPath));
+    REQUIRE(std::filesystem::exists(sharedPath));
 
     // Check that the shared file exists
     std::string syncSharedPath = "faasm://" + fileName;
     REQUIRE(storage::SharedFiles::syncSharedFile(syncSharedPath, "") == 0);
 
     // Flush and check file is gone
     loader.clearLocalCache();
-    REQUIRE(!boost::filesystem::exists(sharedPath));
+
+    // Check the shared files dir exists, but not the specific shared file path
+    REQUIRE(std::filesystem::exists(conf.sharedFilesDir));
```

Comment:
Without the changes in `FileLoader::clearLocalCache()` this chech would fail.

Before, `remove_all(conf.sharedFilesDir)` would delete `conf.sharedFilesDir` as well.

Now, we use `removeAllInside` that recursively deletes the contents of `conf.sharedFilesDir` but not the directory itself.